Orange/data/table.py

```diff
@@ -1465,6 +1465,8 @@ def _compute_contingency(self, col_vars=None, row_var=None):
             arr_indi = [e for e, ind in enumerate(col_indi) if f_cond(ind)]
 
             vars = [(e, f_ind(col_indi[e]), col_desc[e]) for e in arr_indi]
+            if vars and arr is self.X and sp.issparse(self.X):
+                self.X = self.X.tocsc()
```

Comment:
This changes `self.X`, but `arr` still points to the old object right?
Could probably be solved by moving the `if` from L1462 after this and changing to `arr = self.X = self.X.tocsc()`